external/binplacePackages/binplacePackages.depproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ericstj do you think it makes sense to add this support inside of harevestPackages.depproj?